zip-0320.rst

```diff
@@ -81,50 +80,78 @@ Requirements
    address format cannot mistake the address for another address type or
    inadvertently send shielded funds to the address.
 3. No changes to Recipient infrastructure beyond changes to address encoding
-   and decoding should be required as a consequence of this ZIP.
-
-Alternative 1
-=============
+   and decoding should be required as a consequence of this ZIP. In particular,
+   conversion between a transparent-source-only address and the corresponding
+   unrestricted transparent address should be possible using only dependencies
+   that are available to Binance's front-end code.
 
-This alternative was suggested by @hanh in [#hanh-suggestion]_.
```

Comment:
Should we retain this reference?

Reply:
It's already mentioned in the Motivation section on lines 50-52, so this was completely redundant.